lib/class-wp-rest-block-directory-controller.php

```diff
@@ -83,6 +96,57 @@ public function permissions_check() {
 
 		return true;
 	}
+	/* phpcs:enable */
+
+	/**
+	 * Checks whether a given request has permission to install and activate plugins.
```

Comment:
The description of this function is the same as `get_items_permissions_check`. Would be great to detail why/how they're different.

Reply:
The difference is very minimal, and I'm not sure how best to explain, other than the code.. 

It's possible to a user to be able to "get" plugins and potentially be able to install/activate them, but it's only once they actually attempt to install a singular item that the true capability checks can be processed.